src/drive/web/modules/drive/Toolbar/components/Grid/CategoryGridItem.jsx

```diff
@@ -19,7 +16,7 @@ const CategoryGridItem = ({ isSelected, icon, label }) => {
     >
       <div className="u-pos-relative">
         <Icon
-          icon={IconFile}
+          icon={'file-duotone'}
```

Comment:
Nit: `icon='file-duotone'` works equally (no brackets). I wonder why eslint does not fix that automatically here.